packages/EasyWebhook/src/Stores/DoctrineDbalResultStore.php

```diff
@@ -14,18 +14,35 @@
 
 final class DoctrineDbalResultStore extends AbstractDoctrineDbalStore implements ResultStoreInterface
 {
+    /**
+     * @var string
+     */
+    private const DEFAULT_TIMEZONE = 'UTC';
```

Comment:
@evannamon Instead of duplicating this constant, I think we should put it on the `StoreInterface`

Reply:
Yep, moved this constant to `StoreInterface`